porting/build_and_boot/index.rst

```diff
@@ -1,17 +1,9 @@
 Building and booting
 ====================
 
-Having done the necessary research into the specifics of your particular device and prepared a suitable host PC, you are now ready to download and configure the necessary source code and start building.
+Now it's time to download and configure the necessary source code and start building.
 
-This part of the porting process will differ depending on the Halium version you will be building, and which method you choose to follow (in cases where a choice is available). Although the build process for Halium-7.1 and Halium-9.0 carry definite similarities, there are also key differences. Please refer only to the sections that are relevant to the method you will be following!
-
-For the remainder of this guide, information pertaining to Halium-9.0 is also intended to apply to newer versions of Halium. Note that as newer versions evolve, it is quite possible that certain differences will become apparent. As long as such differences remain few and relatively small, new sections will not be added to the guide, but as the number and magnitude differences grow, this option must eventually be chosen. 
-
-There is a separate section describing how to install the build. This section covers both Halium versions. Make sure to note the differences here also.
-
-Furthermore, a section describing an alternate method of building for Halium-9.0 based on Gitlab CI scripts has been included.
-
-The final section is concerned with booting and debugging the boot process.
+This depends on the Halium version and the porting method. Please refer only to the sections that are relevant to the method you are following.
```

Comment:
Re-reading this now, I agree that it may be too long. Most of the time the porter who is familiar with the buide will go straight to the relevant section. These paragraphs were intended for a "first read" to give an outline of this chapter and how to use it. We can shorten it, but I worry that people may mess up when they get to the install section. What do you think? The rest is clearly labeled with the relevant halium version, but that section covers both. Is it sufficiently clear?

Reply:
I've reformulated it a little. pls recheck